src/rgw/rgw_rest.cc

```diff
@@ -1668,6 +1668,12 @@ int RGWDeleteMultiObj_ObjStore::get_params(optional_yield y)
   return op_ret;
 }
 
+int RGWStatBucket_ObjStore::get_params(optional_yield y)
+{
+  report_stats = s->info.args.exists("read-stats");
```

Comment:
`X-Container-Object-Count` and `X-Container-Bytes-Used` are part of the swift api: https://docs.openstack.org/api-ref/object-store/#show-container-metadata

so we need to fetch stats unconditionally for swift

we'll treat that new param as s3-only, so should probably move this override to `RGWStatBucket_ObjStore_S3` and add a new one for `RGWStatBucket_ObjStore_SWIFT` that just sets `report_stats=true`

Reply:
Done.